<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2D Voxel Designer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/2D_Voxel_Designer_Icon.png</normaloff>:/resource/2D_Voxel_Designer_Icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(96, 96, 96);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <property name="leftMargin">
     <number>16</number>
    </property>
    <property name="topMargin">
     <number>32</number>
    </property>
    <property name="rightMargin">
     <number>32</number>
    </property>
    <property name="bottomMargin">
     <number>16</number>
    </property>
    <item>
     <widget class="QWidget" name="verticalWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>90</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resource.qrc">:/resource/2D_Voxel_Designer_LOGO.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNew">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(182, 182, 182);
border:none;
}
QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(200, 200, 200);
border:none;
}
QPushButton:pressed{
color: rgb(0, 0, 0);
background-color: rgb(210, 210, 210);
border:none;
}
QPushButton:disabled {
color: rgb(96, 96, 96);
background-color: rgb(200, 200, 200);
border:none;
}</string>
         </property>
         <property name="text">
          <string>NEW</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOpen">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(182, 182, 182);
border:none;
}
QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(200, 200, 200);
border:none;
}
QPushButton:pressed{
color: rgb(0, 0, 0);
background-color: rgb(210, 210, 210);
border:none;
}
QPushButton:disabled {
color: rgb(96, 96, 96);
background-color: rgb(200, 200, 200);
border:none;
}</string>
         </property>
         <property name="text">
          <string>OPEN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(182, 182, 182);
border:none;
}
QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(200, 200, 200);
border:none;
}
QPushButton:pressed{
color: rgb(0, 0, 0);
background-color: rgb(210, 210, 210);
border:none;
}
QPushButton:disabled {
color: rgb(96, 96, 96);
background-color: rgb(200, 200, 200);
border:none;
}</string>
         </property>
         <property name="text">
          <string>SAVE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widSpace_1" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnModifyImage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(182, 182, 182);
border:none;
}
QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(200, 200, 200);
border:none;
}
QPushButton:pressed{
color: rgb(0, 0, 0);
background-color: rgb(210, 210, 210);
border:none;
}
QPushButton:disabled {
color: rgb(96, 96, 96);
background-color: rgb(200, 200, 200);
border:none;
}</string>
         </property>
         <property name="text">
          <string>IMPORT IMAGE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClearImage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(182, 182, 182);
border:none;
}
QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(200, 200, 200);
border:none;
}
QPushButton:pressed{
color: rgb(0, 0, 0);
background-color: rgb(210, 210, 210);
border:none;
}
QPushButton:disabled {
color: rgb(96, 96, 96);
background-color: rgb(200, 200, 200);
border:none;
}</string>
         </property>
         <property name="text">
          <string>CLEAR IMAGE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widChkWrapper" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chkAspectRatio">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>MAINTAIN ASPECT RATIO</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkLockImage">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>LOCK IMAGE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkShowImage">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>SHOW IMAGE</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widSpace_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widLineColor" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="rightMargin">
           <number>40</number>
          </property>
          <item>
           <widget class="QLabel" name="lblLineColor">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>LINE COLOR</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnLineColor">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnReactorGuide">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(182, 182, 182);
border:none;
}
QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(200, 200, 200);
border:none;
}
QPushButton:pressed{
color: rgb(0, 0, 0);
background-color: rgb(210, 210, 210);
border:none;
}
QPushButton:disabled {
color: rgb(96, 96, 96);
background-color: rgb(200, 200, 200);
border:none;
}
QPushButton:checked{
color: rgb(255, 0, 0);
background-color: rgb(210, 210, 210);
border:none;
}</string>
         </property>
         <property name="text">
          <string>REACTOR GUIDE</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Widget" name="widReactorPreview" native="true">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>210</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>210</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(182, 182, 182);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnResetPoint">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(182, 182, 182);
border:none;
}
QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(200, 200, 200);
border:none;
}
QPushButton:pressed{
color: rgb(0, 0, 0);
background-color: rgb(210, 210, 210);
border:none;
}
QPushButton:disabled {
color: rgb(96, 96, 96);
background-color: rgb(200, 200, 200);
border:none;
}</string>
         </property>
         <property name="text">
          <string>RESET POINT</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Widget</class>
   <extends>QWidget</extends>
   <header location="global">widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
